### ISSUE TITLE
[v2.0.1] Mobile responsive design is broken

### ISSUE THREAD

**Issue by ArtisanByteCrafter** [ironmansoftware/universal-dashboard#380]
It seems the mobile responsive design in v 2.0.1 is broken. I can resize my browser any number of ways and it never adapts. Previous versions have worked as expected.

**Comment by naitmare01** [comment]
I'm having the same issue, any update?

**Comment by adamdriscoll** [comment]
Can you guys give me some sample scripts? For example, this works for me.

```
    New-UDRow -Columns {
        New-UDColumn -SmallSize 12 -LargeSize 4 -Content {
            New-UDCard -Title "Test" -Content {} -BackgroundColor black
        }
        New-UDColumn -SmallSize 12 -LargeSize 4 -Content {
            New-UDCard -Title "Test" -Content {} -BackgroundColor black
        }
        New-UDColumn -SmallSize 12 -LargeSize 4 -Content {
            New-UDCard -Title "Test" -Content {} -BackgroundColor black
        }
    }
```

Small Screen: 

![image](https://user-images.githubusercontent.com/1256531/52710664-21595280-2f4d-11e9-8d3d-2acad9c68890.png)

Large Screen: 

![image](https://user-images.githubusercontent.com/1256531/52710677-2b7b5100-2f4d-11e9-8ca5-c2accf6172ef.png)

**Comment by ArtisanByteCrafter** [comment]
Hi Adam,

I was actually watching your presentation yesterday on UD on Youtube at a PS meetup, and you covered the -Large -Medium and -Small parameters. I had jumped straight from v1.4 to 2.1 and so responsive design used to just magically work, before those parameters were added.

Once I saw how Large, Med, and Small are supposed to work, i tested it and yes it's now indeed working as intended.

 I think this issue can be closed, thank you!!
